tests/redux/actions/storage.test.js

```diff
@@ -97,31 +79,52 @@ describe('Actions.Storage', () => {
         await Actions.setGlobalItem('key', 'value')(store.dispatch, store.getState);
         await Actions.setGlobalItem('excluded', 'not-cleared')(store.dispatch, store.getState);
         await Actions.clear({exclude: ['excluded']})(store.dispatch, store.getState);
-        assert.deepEqual(
-            store.getState().storage.storage,
-            {excluded: 'not-cleared'}
-        );
+        assert.equal(store.getState().storage.storage.excluded.value, 'not-cleared');
+        assert.equal(typeof store.getState().storage.storage.key, 'undefined');
     });
 
     it('rehydrate', async () => {
         const persistor = {
             pause: jest.fn(),
             resume: jest.fn(),
         };
-        await Actions.storageRehydrate({test: '123'})(store.dispatch, persistor);
-        assert.deepEqual(
-            store.getState().storage.storage,
-            {test: '123'}
-        );
-        await Actions.storageRehydrate({test: '456'})(store.dispatch, persistor);
-        assert.deepEqual(
-            store.getState().storage.storage,
-            {test: '456'}
-        );
-        await Actions.storageRehydrate({test2: '789'})(store.dispatch, persistor);
-        assert.deepEqual(
-            store.getState().storage.storage,
-            {test: '456', test2: '789'}
-        );
+        await Actions.storageRehydrate({test: '123'}, persistor)(store.dispatch, store.getState);
+        assert.equal(store.getState().storage.storage.test.value, '123');
```

Comment:
Can we verify the timestamps assigned here?

Reply:
I can, if I mock the date. I'll try to do it. :smiley:

Reply:
Done :)